src/lsst/cmservice/web_app/templates/group_details.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% block title %} Group Details {% endblock %}
 {% block content %}
-<nav class="flex px-2 sm:px-4 lg:px-4" aria-label="Breadcrumb">
+<nav class="flex px-2 sm:px-4 lg:px-4" aria-label="Breadcrumb" xmlns="http://www.w3.org/1999/html">
```

Comment:
why are you explicitly adding the xmlns attribute? This seems weird for a nav bar?

Reply:
Removed